.github/workflows/wheels.yml

```diff
@@ -13,6 +13,10 @@ on:
       - "requirements.txt"
       - "requirements_all.txt"
 
+concurrency:
+  group: wheels
```

Comment:
Add the branch to the group as well so rc does not cancel dev

Reply:
Good point, added 👍